site/load-chart.js

```diff
@@ -95,14 +103,23 @@ function setupChart () {
     }
 
     const node = document.getElementById('servo-chart')
-    const area_dropdown = document.getElementById('selected-area')
+    const test_area_dropdown = document.getElementById('test-selected-area')
+    const date_area_dropdown = document.getElementById('date-selected-area')
```

Comment:
Same comment as the id i.e I would just call this `date_dropdown` or `period_dropdown`